rali/rali/source/rali_api_meta_data.cpp

```diff
@@ -49,6 +49,16 @@ RALI_API_CALL raliCreateCOCOReader(RaliContext p_context, const char* source_pat
 
 }
 
+RaliMetaData
+RALI_API_CALL raliCreateTFReader(RaliContext p_context, const char* source_path, bool is_output){
+    auto context = static_cast<Context*>(p_context);
+    if (!context)
+        THROW("Invalid rali context passed to raliCreateTFReader")
+
+    return context->master_graph->create_tf_record_meta_data_reader(source_path);
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5763058)